### PR TITLE
Enable cluster monitoring in WMCO default namespace.

### PR DESCRIPTION
**Pull request by jfrancoa**
This patch adds the cluster-monitoring label to default WMCO namespace so that endpoints will be created and metrics get generated.

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
@jfrancoa thanks for doing this.

**Comment by jfrancoa** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.14-debug-winc-aws-ipi

File: ci-operator/step-registry/windows/conf/operator/windows-conf-operator-commands.sh [hunk 1]
```diff
@@ -8,6 +8,9 @@ if ! oc get ns "${SUB_INSTALL_NAMESPACE}"; then
   oc create ns "${SUB_INSTALL_NAMESPACE}"
 fi
 
+# Enable monitoring
+oc label namespace "${SUB_INSTALL_NAMESPACE}" openshift.io/cluster-monitoring=true
```
[the remark this answers is not in the document]
Comment (jfrancoa) [reply ci-operator/step-registry/windows/conf/operator/windows-conf-operator-commands.sh:12]: Yes, we can have the 'True' field in our testing, but this ref is being shared along other Prow CI workflows, so it's better to keep the default syntax.

**Comment by jfrancoa** [comment]
From the [rehearsal job logs](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/37699/rehearse-37699-pull-ci-openshift-openshift-tests-private-release-4.14-debug-winc-aws-ipi/1641071811158347776/artifacts/debug-winc-aws-ipi/gather-extra/artifacts/namespaces.json) we can see that cluster monitoring is now enabled:
```

{
            "apiVersion": "v1",
            "kind": "Namespace",
            "metadata": {
                "annotations": {
                    "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"v1\",\"kind\":\"Namespace\",\"metadata\":{\"annotations\":{},\"name\":\"openshift-windows-machine-config-operator\"}}\n",
                    "openshift.io/sa.scc.mcs": "s0:c26,c20",
                    "openshift.io/sa.scc.supplemental-groups": "1000690000/10000",
                    "openshift.io/sa.scc.uid-range": "1000690000/10000"
                },
                "creationTimestamp": "2023-03-29T15:35:24Z",
                "labels": {
                    "kubernetes.io/metadata.name": "openshift-windows-machine-config-operator",
                    "olm.operatorgroup.uid/044dd42c-3ed6-4429-a00d-123faade7f8e": "",
                    "olm.operatorgroup.uid/a08d8ab8-e85d-4c72-b9a3-5a8f2c02c37b": "",
                    "openshift.io/cluster-monitoring": "true",
                    "pod-security.kubernetes.io/enforce": "privileged",
                    "pod-security.kubernetes.io/enforce-version": "v1.24",
                    "security.openshift.io/scc.podSecurityLabelSync": "true"
                },
                "name": "openshift-windows-machine-config-operator",
                "resourceVersion": "33962",
                "uid": "bac6f90b-7f50-48e9-b25d-0cfe47a05c58"
            },
            "spec": {
                "finalizers": [
                    "kubernetes"
                ]
            },
            "status": {
                "phase": "Active"
            }
        },
```
@aravindhp could we approve the PR for merging?

**Comment by aravindhp** [comment]
@jfrancoa I have already approved the PR. I think you need to give the [rehearsal acks](https://github.com/openshift/release/pull/37699#issuecomment-1485236421).

**Pull request merged**